generative_ai/text_generation/test_text_generation_examples.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import gemini_describe_http_image_example
+import gemini_describe_http_pdf_example
+
+
+def test_gemini_describe_http_image_example() -> None:
+    text = gemini_describe_http_image_example.generate_content()
+    assert len(text) > 0
```

Comment:
nit: this is a very imprecise test. Can we at least check for some part of the response which shouldn't change (i.e. puppy, dog, etc.)?

Reply:
True. Goal here is only to do black box testing to verify if both the Google Cloud AI Services & SDK are working. Measuring the quality is out of scope for now.